src/CartManager/AbstractCart.php

```diff
@@ -158,6 +158,10 @@ public function updateItem(string $itemKey, CheckoutableInterface $product, int
             $item->setSubItems($subItems);
         }
 
+        if (!empty($params)) {
+            $item->setCustomProperties($params);
+        }
```

Comment:
$params is always set and always an array. The if shouldn't be necessary.
```suggestion
        $item->setCustomProperties($params);
```